static/src/javascripts/projects/common/modules/userFeatures/cookies/userBenefitsExpiry.ts

```diff
@@ -0,0 +1,17 @@
+import { getCookie } from '@guardian/libs';
+
+export const USER_BENEFITS_EXPIRY_COOKIE = 'gu_user_benefits_expiry';
+
+export const getUserBenefitsExpiryCookie = (): string | null =>
```

Comment:
Small comment but I don't think we need an export on `getUserBenefitsExpiryCookie`, when reviewing I was looking for how this was used outside the scope of this function, but it isn't. 

I'm not too familiar with the linting this repo uses anymore but having an export in front of a function can mean the linter won't flag the function as unused, even if there are no references to it in the codebase, unless of course there's some tool for flagging unused exports, which there may be!

Reply:
Thanks, I've removed this